verilog/tools/ls/autoexpand.cc

```diff
@@ -996,29 +1022,38 @@ std::optional<AutoExpander::Expansion> AutoExpander::ExpandAutoinst(
   }
   const Module &inst_module = modules_.at(type_id);
 
+  // Find an AUTO_TEMPLATE that matches this instance
+  std::optional<absl::string_view> instance_name;
+  if (const auto instance_name_token =
+          GetModuleInstanceNameTokenInfoFromGateInstance(instance)) {
```

Comment:
At this point we're sure this is a `GateInstance`, so it should have an identifier. Wouldn't get parsed otherwise. But after #1788 I don't want to say anything for certain, I guess.

Reply:
Made it so it emits an error if there's no instance name and bails. I don't think it's possible for the name not being there, but at least it won't crash if it happens.